### PR TITLE
Security Shit, Pt. 1. of Pt. 251521 - Return of the Sith

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

You already know what this is about. (Resprites sec shit.)

image

There'll be a part two to this, adding the new Sec Wintercoat, the new Sunglasses, and touching up other alt sec stuff that I couldn't be bothered touching now.

## Why It's Good For The Game

it looks better

## Changelog

:cl:
imageadd: brand new sec jumpsuit, skirt; warden suit, skirt, jacket, hat; HoS suit, skirt, turtleneck, parade jacket (m/f), parade uniform (m/f), wintercoat; security belt mob sprites; security helmet mob sprites.
balance: HoS' wintercoat no longer has a hood, and no longer has more protection then his regular trenchcoat.
imageadd: new sec specific vest
/:cl:

### CODE REVIEW

**Comment by axietheaxolotl** [comment]
screenshots are coming, arm is doing them for me because i am hilariously lazy

**Comment by tralezab** [comment]
he is really lazy

**Comment by Fikou** [comment]
very nice sprites i dont like the edgyness on the base helmet/vest though

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -29,6 +29,9 @@
 	icon_state = "armor"
 	inhand_icon_state = "armor"
 
+/obj/item/clothing/suit/armor/vest/alt/sec
+	icon_state = "armor_sec"
+
```
Comment (Fikou) [inline code/modules/clothing/suits/armor.dm:34]: so... are you gonna give this to sec?

**Comment by Fikou** [comment]
image

**Comment by Fikou** [comment]
image

File: code/modules/clothing/suits/armor.dm (hunk 1 above)
Comment (axietheaxolotl) [reply code/modules/clothing/suits/armor.dm:34]: vsc moment

**Comment by bluetshirtguy** [comment]
The new sprites are cool stuff but I don't really understand the top rows of sprites having a bucket of black paint dumped on their shoulder ¯\_(ツ)_/¯

**Comment by Mothblocks** [comment]
@axietheaxolotl Please fix the screenshot tests

**Comment by thgvr** [comment]
Sec helmet's west dir seems to be shifted one pixel too far to the right. Checked with IDB too. Goggles don't line up with the nose pixel.
image

**Comment by VladinXXV** [comment]
For the most part? Cool shit. But I simply cannot get behind a winter coat with no hood. You monster, the poor HOS is going to freeze his face to death. In all seriousness, what's the point of a winter coat at all if it won't protect you from the cold? If I'm not mistaken, the hood is required for full coverage. And I do not believe any of the existing HOS headwear is insulated against the cold.

**Comment by TheVekter** [comment]
I don't normally bitch about THE SOUL but Sec jumpsuits not being pure blood red just kinda doesn't _feel_ right.

I don't dislike them, I just don't know if they fit.

**Comment by Farquaar** [comment]
You can't take away my man's coat hood

**Comment by axietheaxolotl** [comment]
> I don't normally bitch about THE SOUL

proceeds to bitch about the soul :trollface:

**Comment by IndieanaJones** [comment]
Yeah I don't like the toned-down shade of red either.

**Comment by tralezab** [comment]
> Eh, looks better with all the gear on, actually.

Kinda give me Paranoia vibes with the visors. I'm a fan

**Review by Twaticus — CHANGES_REQUESTED** [review]
icons/mob/clothing/head
swat - left dir misaligned 
helmet - left dir misaligned 
hemlet-flight - left dir misaligned 
helmet-flight-on - left dir misaligned 

icons/mob/clothing/suit
not sure what happened to the back of hostrench but its strange

icons/obj/clothing/suits
missing hoswinter obj icon

icons/mob/clothing/under/security & icons/obj/clothing/under/security.dmi
i dont like the black splotches on the jumpsuit top/bottoms or on the jumpskirt top. just keep it red with the black armband or whatever

**Comment by SkeletalElite** [comment]
The red visor on the helmet is awesome.

**Comment by Victor239** [comment]
What's with the black shorts on the jumpsuits? It looks better and makes more sense for there to be black kneepads instead.

**Comment by SomeoneYouProbablyKnow** [comment]
Gud sprites, my only concern is that dressing sec more like a military will make them act more like a military. But maybe we are ok with that, i just thought i'd bring that up.

**Comment by axietheaxolotl** [comment]
This is still being worked on for the record, I'm just editing some sprites to be better. 👍

**Comment by axietheaxolotl** [comment]
Will re-open when I've done all the changes I want.

**Pull request closed without merging**